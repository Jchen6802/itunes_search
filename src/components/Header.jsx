import styled from 'styled-components';
import React, { Component } from 'react';

const HeaderContainer = styled.div`
    width: 100%;
`;

const AppHeader = styled.header`
    min-height: 130px;
    padding: 15px;
`;

const AppTitle = styled.h1`
    font-size: 1.5em;
`;

export default class Header extends Component {
    render() {
        return <HeaderContainer>
            <AppHeader>
                <AppTitle>
                    Welcome to the iTunes searcher
                </AppTitle>
                <div>Search for all your favorite artists below</div>
            </AppHeader>
        </HeaderContainer>;
    }
}
