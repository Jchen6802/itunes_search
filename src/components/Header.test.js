import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header.jsx';

it('will render as expected', () => {
    const header = renderer.create(
        <Header />
    ).toJSON();

    expect(header).toMatchSnapshot();
});
