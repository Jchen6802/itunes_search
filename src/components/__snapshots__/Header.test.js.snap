// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will render as expected 1`] = `
<div
  className="sc-bdVaJa gZdsYr"
>
  <header
    className="sc-bwzfXH blyfVT"
  >
    <h1
      className="sc-htpNat jBZobA"
    >
      Welcome to the iTunes searcher
    </h1>
    <div>
      Search for all your favorite artists below
    </div>
  </header>
</div>
`;
